sync_diff_inspector/main.go

```diff
@@ -63,7 +63,7 @@ func main() {
 
 	ctx := context.Background()
 	if !checkSyncState(ctx, cfg) {
-		log.Error("check failed!!!")
+		log.Warn("check failed!!!")
```

Comment:
Why change it to Warn?

Reply:
because `Error` will print the stack, looks ugly